descqa/DensityVersusSkyPosition.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import unicode_literals, absolute_import, division
+import os
+import numpy as np
+from scipy.stats import binned_statistic
+from .base import BaseValidationTest, TestResult
+from .plotting import plt
+import healpy as hp
+
+__all__ = ['DensityVersusSkyPosition']
+
+def create_hp_map(ra, dec, nside):
+    """
+    Auxiliary function to generate HEALPix maps from catalogs.
+    It reads the ra and dec in degrees and returns a HEALPix map
+    """
+    pixnums = hp.ang2pix(nside, ra, dec, lonlat=True)
+    return np.bincount(pixnums, minlength=hp.nside2npix(nside)).astype(float)
+
+class DensityVersusSkyPosition(BaseValidationTest):
+    """
+    This tests checks the object density as a function
+    of another map (e.g: extinction, airmass, etc)
+    """
+    def __init__(self,**kwargs): # pylint: disable=W0231
```

Comment:
Why use `**kwargs` here rather than list the kwargs?  It looks like the are all required, so normally that would be good to list explicitly in the function declaration.  Also worth mentioning what the parameters are in the doc string.

Reply:
I did this to be able to "recycle" this script with a different configuration file (and to match the style of most of the tests in `descqa`). @yymao is there an easy way to use positional arguments and be able to read from different config files?

Reply:
@fjaviersanchez I'm not quite sure what you mean "use positional arguments and be able to read from different config files." It is OK to use positional arguments in DESCQA tests; the only requirement is that you must allow additional keyword arguments at the end. See [this test](https://github.com/LSSTDESC/descqa/blob/master/descqa/NumberDensityVersusRedshift.py#L103) for an example. That being said, since these test classes are rarely being called by users in interactive environments, I'm ok with just using `**kwargs`. The more important thing is to specify the arguments in docstring which I see you already did.